mesh/src/test/java/no/nordicsemi/android/mesh/sensorutils/DevicePropertyCharacteristicTest.java

```diff
@@ -180,4 +180,39 @@ public void testDateUTC() {
             counter++;
         }
     }
+
+    @Test
+    public void power() {
+        final ArrayList<Float> expectedSamples = new ArrayList<>();
+        expectedSamples.add(0.0f);
+        expectedSamples.add(0.1f);
+        expectedSamples.add(838860.7f);
+        expectedSamples.add(1677721.4f);
+        expectedSamples.add(null);
+        final ArrayList<byte[]> samples = new ArrayList<>();
+        samples.add(new byte[]{0x00, 0x00, 0x00});
+        samples.add(new byte[]{0x01, 0x00, 0x00});
+        samples.add(new byte[]{(byte) 0xFF, (byte) 0xFF, 0x7F});
```

Comment:
Perhaps the test should include different values for first and second bytes to catch the bug in the code?

Reply:
I have added the suggested case and also padding beginning and end with bytes to test offset.